### ISSUE TITLE
[Feature request+Bug] Sound interpolations

### ISSUE THREAD

**Issue by BKVad1m** [chown2/lunaticvibesf#35]
This feature can set sound interpolations including no interpolation (disable), balance and quality.

**Comment by chown2** [comment]
I don't think I understand, can you rephrase?

**Comment by BKVad1m** [comment]
![image](https://github.com/chown2/lunaticvibesf/assets/76803825/31ee9dc2-25d5-42b4-b71b-0e35e80209ea)

**Comment by BKVad1m** [comment]
https://www.fmod.com/docs/2.00/api/core-api-system.html#fmod_dsp_resampler

**Comment by chown2** [comment]
If I understand correctly, this is about pitch effects "PITCH" and "SPEED" sounding completely awful ("FREQ" work perfectly though) and how that could be approached.
